### DISCUSSION TITLE
I want OpenAIFunction Agent output is a json, what should i do? #872

### DISCUSSION THREAD

**Discussion by Sallery-X** [FlowiseAI/Flowise#923] [Q&A]
<img width="1335" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/45587738/4761edb6-6b19-4db6-8638-6b51147aa68a">
<img width="863" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/45587738/91e5c453-e3d5-4240-a58a-0f82754a7897">
now is a string

**Comment by chungyau97** [comment]
Hi @Sallery-X, 

You may try to write a system message to ask it to return in a JSON code block format.

**Reply by Sallery-X** [reply]
how to do?

**Reply by chungyau97** [reply]
`OpenAI Function Agent` > `Additional Parameters` > `System Message`
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/873c46d6-4d92-4d39-93cc-d7aecdfadbe4)

**Reply by Sallery-X** [reply]
I had try it,  but it doesnt work.

**Reply by Sallery-X** [reply]
could you write a example?
